File: LeetCode-Programs/Detect Capital.py
class Solution:
    def detectCapitalUse(self, word: str) -> bool:
        count_u, count_l=0,0
        first=word[0].isupper()
        for i in range(len(word)):
            if word[i]>='A' and word[i]<='Z':
                count_u+=1
            if word[i]>='a' and word[i]<='z':
                count_l+=1
                
        if count_l==len(word) or count_u==len(word) or (first and count_l==len(word)-1):
            return True
        return False

File: LeetCode-Programs/Detect Capital.cpp
class Solution {
public:
    bool detectCapitalUse(string word) {
        int count_u=0, count_l=0;
        bool first = isupper(word[0]);
        
        for(int i=0;i<word.length();i++){
            if(word[i]<=90 && word[i]>=65)
                count_u++;
       
            if(word[i]>=97 && word[i]<=122)
                count_l++;
        }
        
        if(count_u==word.length() || count_l==word.length() || (first && count_l==word.length()-1))
            return true;
        return false;
    }
};
